### ISSUE TITLE
Make CSV file browse buttons clearer

### ISSUE THREAD

**Issue by wendelap** [OpenEnergyDashboard/OED#383]
It should be clearer that the dashed boxes for uploading CSV files also double as browse buttons. Most websites with this implementation that I've seen say something like "Browse for file or drag and drop here." It doesn't even change your cursor to indicate you can click on it.

**Comment by huss** [comment]
We need to see where this is at as we add CSV input via URL and then fix as appropriate.

**Comment by huss** [comment]
We are about to bring on the ability to load CSV files via a URI request (normally via curl). Given this, the long-term status of the admin drop region for loading CSVs is unclear. We should put this on hold until that is decided.

**Comment by huss** [comment]
The new CSV upload does not have this issue so closing.
